vector/src/main/java/im/vector/app/features/settings/VectorSettingsLabsFragment.kt

```diff
@@ -76,11 +76,11 @@ class VectorSettingsLabsFragment :
         }
 
         findPreference<VectorSwitchPreference>(VectorPreferences.SETTINGS_LABS_NEW_APP_LAYOUT_KEY)?.let { pref ->
-            pref.isVisible = vectorFeatures.isNewAppLayoutEnabled()
+            pref.isVisible = vectorFeatures.isNewAppLayoutFeatureEnabled()
         }
 
         findPreference<VectorSwitchPreference>(VectorPreferences.SETTINGS_LABS_UNREAD_NOTIFICATIONS_AS_TAB)?.let { pref ->
-            pref.isVisible = !vectorFeatures.isNewAppLayoutEnabled()
+            pref.isVisible = !vectorFeatures.isNewAppLayoutFeatureEnabled()
```

Comment:
same, we want to check the labs flag here.

Reply:
What we should do probably here is make it visible depending on the feature flag, and make it enabled depending on the labs flag